server/src/main/java/com/showmetheroapes/paracord/services/StrandService.java

```diff
@@ -4,16 +4,26 @@
 import com.showmetheroapes.paracord.models.dto.StrandDTO.Request;
 import com.showmetheroapes.paracord.repositories.StrandRepository;
 import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
 
 @Service
 public class StrandService {
+  private static final Logger logger = LoggerFactory.getLogger(StrandService.class);
+
   private StrandRepository strandRepository;
+  private RestTemplate restTemplate;
 
   @Autowired
-  public StrandService(StrandRepository strandRepository) {
+  public StrandService(StrandRepository strandRepository, RestTemplate restTemplate) {
     this.strandRepository = strandRepository;
+    this.restTemplate = restTemplate;
```

Comment:
So I'm confused how this rest template is connected to the rest template in the `CronJobConfiguration`.. please help meeeeee

Reply:
This would be easier to explain in person